src/ucp/tag/offload.c

```diff
@@ -556,9 +557,14 @@ ucs_status_t ucp_tag_offload_sw_rndv(uct_pending_req_t *self)
     rndv_rts_hdr = ucs_alloca(rndv_hdr_len);
     packed_len   = ucp_tag_rndv_rts_pack(rndv_rts_hdr, req);
 
-    return uct_ep_tag_rndv_request(ep->uct_eps[req->send.lane],
-                                   req->send.msg_proto.tag.tag,
-                                   rndv_rts_hdr, packed_len, 0);
+    status = uct_ep_tag_rndv_request(ep->uct_eps[req->send.lane],
+                                     req->send.msg_proto.tag.tag, rndv_rts_hdr,
+                                     packed_len, 0);
+    if (ucs_unlikely(status != UCS_OK)) {
+        return status;
+    }
+
+    return UCS_OK;
```

Comment:
what is the difference with just returning status? Also, should not we return OK in case of error here?

Reply:
leftover
yes, we should - will do it in a separate PR

Reply:
why not here?  You already have a single func to handle status - one line change

Reply:
> why not here? You already have a single func to handle status - one line change

I've removed my previous changes in the offload code. So, checking the status will be an unrelated change.